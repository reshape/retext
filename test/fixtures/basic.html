<article class="my-article">
  <h1>Hello "world"...</h1>
  <p>The three wise monkeys [. . .] sometimes called the three mystic
  apes--are a pictorial maxim. Together they embody the proverbial
  principle to ("see no evil, hear no evil, speak no evil"). The
  three monkeys are Mizaru (:see_no_evil:), covering his eyes, who
  sees no evil; Kikazaru (:hear_no_evil:), covering his ears, who
  hears no evil; and Iwazaru (:speak_no_evil:), covering his mouth,
  who speaks no evil.</p>
</article>
