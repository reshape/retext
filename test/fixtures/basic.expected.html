<article class="my-article">
  <h1>Hello “world”…</h1>
  <p>The three wise monkeys […] sometimes called the three mystic
  apes—are a pictorial maxim. Together they embody the proverbial
  principle to (“see no evil, hear no evil, speak no evil”). The
  three monkeys are Mizaru (🙈), covering his eyes, who
  sees no evil; Kikazaru (🙉), covering his ears, who
  hears no evil; and Iwazaru (🙊), covering his mouth,
  who speaks no evil.</p>
</article>
